src/main/java/step4/domain/Car.java

```diff
@@ -0,0 +1,45 @@
+package step4.domain;
+
+public class Car {
+    private final int condition = 4 ;
+    Name name;
+    Position position;
+
+    public Car(String name) {
+        this(new Name(name), new Position());
+    }
+
+    public Car(Name name, Position position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public void move(int randomValue) {
+        if(condition < randomValue){
+            position.plus();
+        }
+    }
+
+    public boolean isMoving() {
+        return myPosition() > 0;
```

Comment:
```suggestion
    public boolean movable() {
        return myPosition() > 0;
```

부족한 영어 실력이지만 엘레강트 오브젝트의 포비님 영상에 따라,
boolean 을 리턴하는 메서드는 형용사로 지을 수 있다는 점을 언급해보았습니다.
굳이 수정하지 않으셔도 됩니다.